tfhe/src/shortint/parameters/list_compression.rs

```diff
@@ -10,13 +10,13 @@ use tfhe_versionable::Versionize;
 #[derive(Copy, Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize, Versionize)]
 #[versionize(CompressionParametersVersions)]
 pub struct CompressionParameters {
-    pub br_level: DecompositionLevelCount,
```

Comment:
I would prefer we don't do such changes notably for serilization formats for which field orders may matter, it means we have to update the CompressionParametersVersions for no gains